src/super_gradients/wandb/artifacts.py

```diff
@@ -0,0 +1,13 @@
+import os
+import wandb
```

Comment:
Please use the same import logic we used in the WandbLogger 
```
try:
    import wandb
except (ModuleNotFoundError, ImportError, NameError):
    pass  # no action or logging - this is normal in most cases
```
then check if `wandb is not None`